src/main/java/frc/robot/Robot.java

```diff
@@ -156,50 +156,35 @@ private void configureBindings() {
               }
             }));
 
-    hardware
-        .driverController
-        .rightTrigger()
-        .onTrue(robotCommands.confirmShotCommand())
-        .onFalse(robotCommands.stopShootingCommand());
-    hardware
-        .driverController
-        .leftTrigger()
-        .onTrue(robotCommands.intakeCommand())
-        .onFalse(robotCommands.stowCommand());
+    hardware.driverController.rightTrigger().onTrue(robotCommands.confirmShotCommand());
```

Comment:
need to tell robotmanager when the confirm shot mode is active/inactive. probably by just extending the ontrue/onfalse here